api/v2/clairpb/clair.proto

```diff
@@ -117,8 +125,6 @@ message DeleteNotificationRequest {
 	string name = 1;
 }
 
-
-
 service AncestryService{
```

Comment:
I cannot comment on the line, but we should rename "DeleteNotification" to "MarkNotificationAsRead" and make HTTP DELETE be an implementation detail of the rest gateway